paddle/math/tests/test_matrixCompare.cpp

```diff
@@ -181,7 +196,7 @@ void testMatrixInverse(int height) {
   cpu->add(*outputCheck);
 
   gpu->copyFrom(*cpu);
-  cpu->inverse(cpuI, false);
+  cpu->inverse(cpuI, true);
   gpu->inverse(gpuI, false);
```

Comment:
Inverse also has the same problem with transpose when memAlloc is true. I want test here.